eng/Versions.props

```
@@ -49,7 +49,7 @@
     <!--
       TODO: Remove pinned version once arcade supplies a compiler that enables the repo to compile.
     -->
-    <MicrosoftNetCompilersToolsetVersion>4.4.0-1.22356.12</MicrosoftNetCompilersToolsetVersion>
+    <MicrosoftNetCompilersToolsetVersion>4.4.0-1.22357.16</MicrosoftNetCompilersToolsetVersion>
```

```suggestion
    <MicrosoftNetCompilersToolsetVersion>4.4.0-1.22358.14</MicrosoftNetCompilersToolsetVersion>
```

points to https://github.com/dotnet/roslyn/commits/40f3783cda2a10e70a280298d83f3e52da254129, that has the fix (https://github.com/dotnet/roslyn/commit/601d6e89bd7405d4e07c467cc383bbc81ca7eebe).